grafana-plugin/src/utils/utils.ts

```diff
@@ -18,18 +18,18 @@ export class KeyValuePair<T = string | number> {
   }
 }
 
+export const formatBackendError = (payload: string | Record<string, unknown>) =>
+  typeof payload === 'string'
+    ? payload
+    : Object.keys(payload)
```

Comment:
actually this code doesn't always provide human readable text, just note, let's fix it a bit later